observability/logging/logging-6.0/log6x-output.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-output-types"]
+= Log output types
+:context: logging-output-types
+
+toc::[]
+
+Outputs define the destination where logs are sent to from a log forwarder. You can configure multiple types of outputs in the `ClusterLogForwarder` custom resource (CR) to send logs to servers that support different protocols.
+
+include::modules/log6x-supported-log-outputs.adoc[leveloffset=+1]
+
+[id="log6x-output-types-descriptions"]
+== Output type descriptions
+
+`default`:: The on-cluster, Red{nbsp}Hat managed log store. You are not required to configure the default output.
```

Comment:
This is not correct in Logging 6.0. In Logging 6.0 is not more valid the word "default", then, it can't be used as not referencing anything and not logs sent to the Red Hat Managed Log Store. 

For being able to log forward to the Red Hat Managed Log store, it's needed to define the output where the URL is the Red Hat Managed Log store as per upstream docs: https://github.com/openshift/cluster-logging-operator/blob/f1b017aafb51c6d2cd2eb8da405c10d5d61cf979/docs/administration/upgrade/to_v6.0.adoc?plain=1#L281

Reply:
Removed Line 16 and the Note associated with it on Line 18.